### DISCUSSION TITLE
[Bug] Stop at a specific event

### DISCUSSION THREAD

**Discussion by heparidayo** [MaaAssistantArknights/MaaAssistantArknights#4969] [Q&A] [answered]
### My environment
Korean client
Windows 11 Pro    

### Bug Description
Phantom & Crimson Solitaire   
When i encounter the event(attached screenshot) mma stops.    
If I manually hand over the event, it will proceed normally again.    

![StoppedImage1](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/107000683/541ef4a3-89de-4c68-bfcb-7cfac5c785f3)
![StoppedImage2](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/107000683/b3cfbead-eb4f-4cdc-b847-8195758343db)

**Comment by ABA2396** [comment, marked as answer]
Post the log here #4973

**Reply by heparidayo** [reply]
oops, thank you

**Reply by ABA2396** [reply]
Under the `debug` folder, there will be an `asst.log` file and a `roguelike` folder. Remember to send them.

**Reply by heparidayo** [reply]
My log and folder here.   
[logs.zip](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/11584848/logs.zip)

Should I upload it to #4973 too?

**Reply by ABA2396** [reply]
I have already forwarded the files for you.
